grails-app/controllers/org/pih/warehouse/core/LocationController.groovy

```diff
@@ -112,7 +112,12 @@ class LocationController {
                     if (locationInstance?.id == session?.warehouse?.id) {
                         session.warehouse = locationInstance
                     }
-                    flash.message = "${warehouse.message(code: 'default.updated.message', args: [warehouse.message(code: 'location.label', default: 'Location'), locationInstance.id])}"
+                    flash.message = [
+                        [code: 'default.updated.message', args: [warehouse.message(code: 'location.label', default: 'Location'), locationInstance.id]]
+                    ]
+                    if (locationInstance.supportedActivities.contains(ActivityCode.REQUEST_APPROVAL.id)) {
+                        flash.message.add([code: 'location.supportedActivities.noRequestApprovers', args: [g.createLink(controller: "user", action: "list")]])
+                    }
```

Comment:
The comma isn't necessary anymore since the message is rendered in the new line like
![image](https://github.com/openboxes/openboxes/assets/35233347/7a0929b0-b336-46af-b3d5-5c8e359a1518)

Previously @kchelstowski must have added the comma to separate both messages since they were rendered in the same line.